Client/AppDelegate.swift

```diff
@@ -26,6 +26,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 
 struct SDKConfig: KarhooSDKConfiguration {
+    func requestNewAuthenticationCredentials(callback: @escaping () -> Void) {
+        callback()
```

Comment:
This may need further implementation down the line, i.e. actually refreshing the access token like a real DP would do.